mackup/mackup.py

```diff
@@ -36,13 +36,13 @@ def check_for_usable_environment(self):
                 " Run mackup --help for guidance."
             )
 
-        # Do we have a folder to put the Mackup folder ?
+        # Do we have a folder to put Mackup into?
```

Comment:
It's really not Mackup that we put, but the Mackup data files. Can you reword to make it more explicit?

Reply:
> make it more explicit?

You’re right – let me know if c638fd8 doesn’t go far enough.